Makefile

```diff
@@ -4,6 +4,9 @@ build:
 
 format:
 	gofmt -s -w .
+	
+	go get -u golang.org/x/tools/cmd/goimports
+	goimports -local github.com/elastic/elastic-package/ -w .
```

Comment:
Sorry, but I just found this on https://godoc.org/golang.org/x/tools/cmd/goimports:

> In addition to fixing imports, goimports also formats your code in the same style as gofmt so it can be used as a replacement for your editor's gofmt-on-save hook.

So maybe we don't even need `gofmt -s -w .` any more?

Reply:
Right, it includes the `gofmt` behavior. Working on this

Reply:
Fixed!